services/aur/aur.service.js

```diff
@@ -66,17 +67,15 @@ class AurLicense extends BaseAurService {
   static get route() {
     return {
       base: 'aur/license',
-      format: '(.+)',
-      capture: ['pkg'],
+      pattern: ':packageName',
```

Comment:
`packageName` for consistency with other badges (and because `package` is reserved).

Reply:
Makes sense. This one used `package` in the examples and `pkg` in `capture`. It'll be simpler to use one or the other, and when we're building a form in the frontend, I suppose **package name** seems nicer than **pkg** which is why I went with that.